qiskit/algorithms/minimum_eigensolvers/__init__.py

```diff
@@ -34,12 +37,18 @@
    NumPyMinimumEigensolverResult
    VQEResult
    AdaptVQEResult
+   SamplingMinimumEigensolverResult
+   SamplingVQE
```

Comment:
This is the algorithm right, it should not be here in this list too, just the results